tests/functional/tests/backup.py

```diff
@@ -0,0 +1,35 @@
+"""Generic setup for functional tests."""
+import logging
+import os
+import unittest
+
+from cou.steps.backup import backup
+from cou.zaza_utils import clean_up_libjuju_thread
+
+logger = logging.getLogger(__name__)
+
+
+class BackupTest(unittest.TestCase):
+    """Code for validating Zaza test class loader."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run class setup for noop tests."""
+        super(BackupTest, cls).setUpClass()
+
+    def tearDown(self):
+        """Close thread."""
+        logger.info("Closing libjuju thread.")
+        clean_up_libjuju_thread()
+        logger.info("Removing test file")
+        try:
+            os.remove(self.backup_file)
```

Comment:
Why we need to use exception here? I think removing it if exist would be more than enough.

Reply:
easier

Reply:
Yes, but like this you are ignoring other error exception also, you should use `FileNotFoundError`.